conftest.py

```diff
@@ -76,46 +80,81 @@ def sample_dag():
         session.query(TI).delete()
 
 
+def populate_table(path: str, table: Table, hook: BaseHook) -> None:
+    """
+    Populate a csv file into a sql table
+    :param path: path to csv file
```

Comment:
```suggestion
    Populate a csv file into a sql table

    :param path: path to csv file
```

We need a blank line between docstring description/summary and the param docs

Reply:
This is needed by Sphinx